extensions/blocks/podcast-player/components/podcast-player.js

```diff
@@ -88,6 +98,11 @@ export class PodcastPlayer extends Component {
 	 */
 	handleError = () => {
 		this.setState( { playerState: STATE_ERROR } );
+
+		speak(
+			`${ sprintf( __( 'Error: Episode unavailable. (Open in a new window)', 'jetpack' ) ) }`,
```

Comment:
```suggestion
			`${ sprintf( __( 'Error: unable to play this Episode. Click to open in a new window', 'jetpack' ) ) }`,
```

I'm thinking "unavailable" isn't correct. It's more "it couldn't play".

Also would the context of "Open in a new window" be obvious enough? 

Also is "click" universal?

Reply:
I was trying to utilize the same text as being used in the real error message. I watched a screen reader user in a usability test say that they would do a kind of Find on page after aria-live announcements for that text. So, I wanted the error to match so that people who use that technique would be able to find the error area on the page so that they could open the track in a new window, if available.

Another technique you can use is to utilize the Rotor to find links. So, I would prefer our error message link be more descriptive instead of just `(Open in a new window)`. Apparently it says `(Open in a new tab)` in the error message though, so I'd need to change that. 

![link-rotor](https://user-images.githubusercontent.com/967608/78033946-b45d7380-732c-11ea-83c4-872791609079.gif)


What would make it more accessible is to:
- change the visual error message to have the link be more descriptive outside of visual context
- change the announced message to match the visual error message

Reply:
I think I'll try a visually hidden title in the link to give more context, like `Open <span class="visually-hidden">[episode title]</span> in a new tab`.

Reply:
Rather than interpolate in the translated string with a `<span>` (which doesn't seem doable without a large amount of extra work), I added a visually hidden span before the Open in a new tab text so that in the Rotor it shows up as "[episode title]: Open in a new tab" and changed the announced error text to say `tab` instead of `window` so that the messages match.